dockerfiles/Dockerfile.coconut_develop

```diff
@@ -0,0 +1,33 @@
+# This install is NOT minimal!
+#   TODO: Minimize the docker container
+FROM debian:8.5
+MAINTAINER x10an14
+
+#  'apk add'
+#     on alpine
+#=> equivalent of
+#  'apt-get install'
+#     on debian/ubuntu
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git \
+    gcc \
+    g++ \
+    make \
+    python3 \
+    python3-dev \
+    python3-pip
+
+RUN ln -sf /usr/bin/pip3 /usr/bin/pip && ln -sf /usr/bin/python3 /usr/bin/python
+
+# Copy source-code into docker image
+ADD . /usr/coconut
+WORKDIR /usr/coconut
+
+# install
+RUN make dev
+
+# Open a console with current (source-code)
+# version of Coconut
+ENTRYPOINT ["python3", "coconut/main.py"]
```

Comment:
@hasufell I don't really know docker files, so thus haven't really looked at this PR since it was made (and now it's been stale for a while). If you're up for it, it'd be awesome if you were able to revive this.